packages/main/src/plugin/appearance-init.ts

```diff
@@ -47,6 +47,12 @@ export class AppearanceInit {
           default: 0,
           step: 0.1,
         },
+        [`${AppearanceSettings.SectionName}.${AppearanceSettings.NavigationAppearance}`]: {
+          description: 'Select icon and title or just icon for navigation icons',
+          type: 'string',
+          enum: ['icon + title', 'icon'],
+          default: import.meta.env.DEV ? 'icon + title' : 'icon',
```

Comment:
nitpick: use constants for 'icon' and 'icon + title', so in case we change the enum, it'll automatically change the default value